doc/whatsnew/2/2.15/index.rst

```diff
@@ -0,0 +1,50 @@
+***************************
+ What's New in Pylint 2.15
+***************************
+
+.. toctree::
+   :maxdepth: 2
+
+:Release: 2.14
+:Date: TBA
+
+Summary -- Release highlights
+=============================
+
+New checkers
+============
+- ...
+
+
+Removed checkers
+================
+- ...
+
+
+
+Extensions
+==========
+- ...
+
+
+Bug fixes:
+==========
+- ...
+- ...
+
+False positives fixed
+=====================
+- ...
+
+False negatives fixed
+=====================
+- ...
```

Comment:
I like this, but I would put Bug fixes third and call it "Other bug fixes" to help new contributors.